lib/ansible/modules/pip.py

```diff
@@ -5,7 +5,6 @@
 
 from __future__ import annotations
 
-
```

Comment:
Try to avoid sneaking formatting-only changes into PRs. They tend to clutter the diff and let important things go unnoticed. They are also unrelated to the PR goal, making it non-atomic. It is a bad tone to mix formatting/refactoring/functional changes. They all should be separate because otherwise, it becomes difficult to distinguish purposes of each included change. It may also cause additional headache when backporting due to increased likelihood of having conflicts while cherry-picking.